pkg/controller/stacks/stack/stack.go

```diff
@@ -160,29 +175,157 @@ func (h *stackHandler) update(ctx context.Context) (reconcile.Result, error) {
 	return reconcile.Result{}, nil
 }
 
-func (h *stackHandler) createNamespacedRoleBinding(ctx context.Context, owner metav1.OwnerReference) error {
-	cr := &rbac.Role{
+func (h *stackHandler) createPersonaClusterRoles(ctx context.Context, labels map[string]string) error {
+	// TODO(displague) refine 'editor' persona roles
+	for persona, verbs := range map[string][]string{
+		"admin": {"get", "list", "watch", "edit", "create", "delete"},
+		"edit":  {"get", "list", "watch", "edit"},
+		"view":  {"get", "list", "watch"},
+	} {
+		rules := []rbacv1.PolicyRule{}
+		for _, crd := range h.ext.Spec.CRDs {
+			rules = append(rules, rbacv1.PolicyRule{
+				APIGroups: []string{crd.GroupVersionKind().Group},
+				Resources: []string{crd.Kind},
+				Verbs:     verbs,
+			})
+		}
+		name := fmt.Sprintf(clusterRoleNameFmt, h.ext.GetNamespace(), h.ext.GetName(), h.ext.Spec.Version, persona)
+
+		namespaceToCrossplaneScopes := map[bool]string{
```

Comment:
I might just change this to an because I think it's clearer for me

```
if h.isNamespaced() {
    scope = namespaced
else {
    scope = environment
}